eslint.config.mjs

```diff
@@ -58,7 +58,7 @@ export default tsEslint.config([
       "sonarjs/no-element-overwrite": "error",
       "sonarjs/no-identical-conditions": "error",
       "sonarjs/no-identical-expressions": "error",
-      "@ubiquity-os/empty-string-checker": "warn",
+      "@ubiquity-os/no-empty-string": "warn",
```

Comment:
I realize in the context of the other rule names, this is actually best to be called `@ubiquity-os/no-empty-strings` plural.